src/api/commands/address/AddressListCommand.ts

```diff
@@ -38,13 +40,18 @@ export class AddressListCommand extends BaseCommand implements RpcCommandInterfa
     @validate()
     public async execute( @request(RpcRequest) data: RpcRequest, rpcCommandFactory: RpcCommandFactory): Promise<Bookshelf.Collection<Address>> {
         const profileId = data.params[0];
-        if ( !profileId ) {
+        if (!profileId) {
             // Get all addresses.
             return this.addressService.findAll();
         }
 
```

Comment:
profileId is not optional, if there's no profile id, throw error.

Reply:
Then this test must be fixed - https://github.com/particl/particl-market/blob/develop/test/black-box/address/AddressRemoveCommand.test.ts#L64

Reply:
Then we need to fix this test https://github.com/particl/particl-market/blob/develop/test/black-box/address/AddressRemoveCommand.test.ts#L64

Reply:
yes, remove that and you can add test to make sure address list returns the addresses for the correct profile

Reply:
I'll simply add addressId as arg there, so that it will return empty res.